<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseProjectClass</class>
 <widget class="QMainWindow" name="CourseProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CourseProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="carsLW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Add new car</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Drivetrain setup</string>
     </property>
     <widget class="QRadioButton" name="rwdRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Rear-wheel drive</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="fwdRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Front-wheel drive</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="awdRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>All-wheel drive</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Fuel</string>
     </property>
     <widget class="QRadioButton" name="petrolRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Petrol</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="gasPetrolRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Gas-petrol</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="hybridRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hybrid</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="electricityRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Electricity</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>$</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="priceSP">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>40000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>20000</number>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Transmission type</string>
     </property>
     <widget class="QRadioButton" name="mechanicRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mechanic</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="automaticRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Automatic</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="modelLE">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="firmsCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bodyTypesCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>111</height>
      </rect>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="editBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewServicesBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>View Services...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CourseProject.qrc"/>
 </resources>
 <connections/>
</ui>
